packages/ibm-products/src/components/Card/Card.js

```diff
@@ -259,16 +279,32 @@ export let Card = forwardRef(
     });
     return (
       <div {...getCardProps()}>
-        {media && <div className={`${blockClass}__media`}>{media}</div>}
+        {!getStarted && media && (
+          <div className={`${blockClass}__media`}>{media}</div>
+        )}
         {Pictogram && (
           <div className={`${blockClass}__pictogram`}>
             <Pictogram />
           </div>
         )}
+        {getStarted && sequence && (
+          <div className={`${blockClass}__sequence`}>{sequence}</div>
+        )}
+        {getStarted && status && (
+          <div className={`${blockClass}__status`}>
+            {status === 'incomplete' ? <Incomplete /> : ''}
```

Comment:
two things

1. you should be able to shorten these by just using the `&&` notation so you don't need the empty strings.
2. i see that there's a `none` value for status to indicate a null or empty state. i think you can forgo that and just make the status either `complete` or `incomplete`